torchvision/transforms/_presets.py

```diff
@@ -25,6 +25,12 @@ def forward(self, img: Tensor) -> Tensor:
             img = F.pil_to_tensor(img)
         return F.convert_image_dtype(img, torch.float)
 
+    def __repr__(self) -> str:
+        return self.__class__.__name__ + "()"
+
+    def describe(self) -> str:
```

Comment:
This is the "human-readable" version. Inspired from Pandas' `describe()` method which summarizes dataframes. It provides a high-level description of what the transform is actually doing.